projects/sandbox/export/export/snapshotter.py

```diff
@@ -57,6 +56,7 @@ def __init__(
         length += kernel_length + fduration
         self.size = int(length * sample_rate)
 
+        fftlength = kernel_length + fduration
```

Comment:
I like this idae. I think it might be better practice to keep `fftlength` as an argument to our script, annotate it as `None`, and then calculate the default if it is not passed.
```python
@scriptify
def main(
    ...
    fftlength: Optional[float] = None,
    ...
):
    fftlength = fftlength or kernel_length + fduration
```
This keeps the flexibility of passing it as an argument, and also lets the user know in the function signature that this is a parameter of the calculation. 

Up to you though, either way will work

Reply:
Yeah, I like that.